modules/rest-util/test/blaze/middleware/fhir/output_test.clj

```diff
@@ -205,5 +227,47 @@
       [:headers "Content-Type"] := "application/fhir+xml;charset=utf-8"
       [:body parse-xml :issue 0 :diagnostics] := "Invalid white space character (0x1e) in text to output (in xml 1.1, could output as a character entity)")))
 
+(deftest binary-resource-test
+  (testing "possible accept headers"
+    (are [accept content-type body]
+         (given (call (binary-resource-handler-200 {:content-type "text/plain" :data "MTA1NjE0Cg=="}) {:headers {"accept" accept}})
+           :status := 200
+           [:headers "Content-Type"] := content-type
+           [:body bs/from-byte-array] := body)
+      "application/fhir+json" "application/fhir+json;charset=utf-8" #blaze/byte-string"7B2264617461223A224D5441314E6A453043673D3D222C22636F6E74656E7454797065223A22746578742F706C61696E222C227265736F7572636554797065223A2242696E617279227D"
```

Comment:
I would not byte-compare the FHIR output. please use `parse-json` and `parse-xml` like in the other tests

Reply:
saying that, please just write two `testing` blocks, one for JSON and one for XML